packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -612,6 +612,8 @@ export async function setupServer(
       host: serverOptions.host,
       open: serverOptions.open,
       headers: serverOptions.headers,
+      // Disable the websocket if live reload is disabled (false/undefined are the only valid values)
+      ws: serverOptions.liveReload === false ? false : undefined,
```

Comment:
Shouldn't the check be updated to also account for `serverOptions.hmr` ie

`serverOptions.liveReload === false && serverOptions.hmr === false`